apps/rush-lib/src/api/Rush.ts

```diff
@@ -19,40 +45,54 @@ export class Rush {
   /**
    * This API is used by the `@microsoft/rush` front end to launch the "rush" command-line.
    * Third-party tools should not use this API.  Instead, they should execute the "rush" binary
-   * and start a new NodeJS process.
+   * and start a new Node.js process.
    *
    * @param launcherVersion - The version of the `@microsoft/rush` wrapper used to call invoke the CLI.
-   * @param isManaged - True if the tool was invoked from within a project with a rush.json file, otherwise false. We
-   *  consider a project without a rush.json to be "unmanaged" and we'll print that to the command line when
-   *  the tool is executed. This is mainly used for debugging purposes.
    */
```

Comment:
Add a `@remarks` note saying that earlier Rush front-ends used a different contract.  It's supported, but undocumented.